worker/src/test/java/org/apache/celeborn/service/deploy/worker/storage/PartitionFilesSorterSuiteJ.java

```diff
@@ -102,15 +99,12 @@ public void prepare(boolean largefile) throws IOException {
       while (buf2.hasRemaining()) {
         channel.write(buf2);
       }
+      partitionSize[mapId] = partitionSize[mapId] + batchHeader.length + mockedData.length;
     }
     originFileLen = channel.size();
     fileInfo.getChunkOffsets().add(originFileLen);
-    fileInfo.updateBytesFlushed((int) originFileLen);
```

Comment:
that's why the `testLargeFile` failed before.  the created file is larger than 2GB, which is bigger than what `Integer.MAX_VALUE` can handle. cc @waitinfuture

Reply:
Good catch! Since `dataSize` is random, it happens randomly that the file exceeds 2G.

Reply:
Although IMO, in reality it will not happen because `updateBytesFlushed` is passed with the length of `CompositeByteBuf#readableBytes`, which is of `int` type. :)